src/docs/build-posix.dox

```diff
@@ -14,6 +14,8 @@ To build from the WiredTiger GitHub repository requires
 <a href="http://www.swig.org/">SWIG</a> (if building to support the Python API).
 We also suggest <a href="https://ninja-build.org/">Ninja</a> and
 <a href="https://ccache.dev/">ccache</a> for improved incremental build times.
+If wanting to compile the Python API, we also require the following python3-dev and swig packages.
```

Comment:
I think the existing paragraph already contains most of the information:

_To build from the WiredTiger GitHub repository requires [git](http://git-scm.com/), [GCC 8.5 or later](https://gcc.gnu.org/) or [Clang 7.01 or later](https://clang.llvm.org/), and [CMake 3.10.0 or later](https://cmake.org/), and [Python 3 or later](https://python.org/) (if building to support the Python API), and [SWIG](http://www.swig.org/) (if building to support the Python API). We also suggest [Ninja](https://ninja-build.org/) and [ccache](https://ccache.dev/) for improved incremental build times._

I think we should simply mention the `python3-dev` where we mention Python 3. (The beginning of the first sentence looks weird by the way). I suggest:

_To build from the WiredTiger GitHub repository, the following is required:
[Git](http://git-scm.com/), [GCC 8.5 or later](https://gcc.gnu.org/) or [Clang 7.01 or later](https://clang.llvm.org/), and [CMake 3.10.0 or later](https://cmake.org/)._

_To support the Python API, [Python 3 or later](https://python.org/), [SWIG](http://www.swig.org/) and \c python3-dev are required._

_We also suggest [Ninja](https://ninja-build.org/) and [ccache](https://ccache.dev/) for improved incremental build times._